docs/extend/testing.md

```diff
@@ -75,13 +81,32 @@ mage docker:composeBuild
 # Bring up all containers, wait until they are healthy, and put them in the background.
 mage docker:composeUp
 
+# Build the test binary
+mage buildSystemTestBinary
+
 # Run all system and integration tests.
-INTEGRATION_TESTS=1 pytest ./tests/system
+INTEGRATION_TESTS=1 \
+BEAT_STRICT_PERMS=false \
+ES_PASS="testing" \
+pytest ./tests/system
 
 # Stop all started containers.
 mage docker:composeDown
 ```
 
+To run a test from `x-pack/`, you need to set the `MODULES_PATH` to the full
+path to the `x-pack` modules directory:
```

Comment:
That's not completely true, this only applies to the modules tests, there are other system tests that do not require setting `MODULES_PATH`.

```suggestion
To run a module test from `x-pack/`, you need to set the `MODULES_PATH` to the full
path to the `x-pack` modules directory:
```